oca_dependencies.txt

```diff
@@ -1,2 +1,3 @@
 connector
-laslabs-stock https://github.com/laslabs/odoo-stock.git
+stock-logistics-workflow https://github.com/laslabs/stock-logistics-workflow.git feature/9.0/LABS-187-migrate-connectoreasypost-to-oca
```

Comment:
Add the fact that this is not using OCA/9.0 to Readme and note the PRs that should be monitored in order to do so.